src/restd.c

```diff
@@ -61,7 +64,7 @@ restd(f77int *lun, f77int *tddesc, f77int *nctddesc, f77int *ctddesc)
 /*
 **          desc is a local descriptor.
 */
-            numtbd( lun, &desc, nemo, &tab, &ictbd, 9, 1 );
+            numtbd_f( *lun, desc, nemo, NEMO_STR_LEN, &tab, &ictbd );
```

Comment:
Suggest you remove the spaces after and before parens. Not usual in C code.

Reply:
OK will do.